validation/src/main/scala/hmda/validation/rules/lar/validity/V338.scala

```diff
@@ -0,0 +1,27 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+
+object V338 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V338"
+
+  def apply(lar: LoanApplicationRegister): Result = {
```

Comment:
This needs an `override` modifier. @hkeeler pointed this out, it's very inconsistent throughout the codebase. Both `name` and `apply` override from base abstract class, and should have this notation. There is a fix for all previous inconsistencies in another PR